framework/assets/yii.validation.js

```diff
@@ -36,7 +36,7 @@ yii.validation = (function ($) {
             }
         },
 
-        'boolean': function (value, messages, options) {
+        boolean: function (value, messages, options) {
```

Comment:
@nkovacs Got it. Thanks for explanation. Should we leave it as is in this case to support ES3?